src/models/Chain.js

```diff
@@ -33,3 +33,11 @@ export type ChainRecord<T> = {|
   xdai?: T,
   ethereum?: T,
 |};
+
+// Based on: https://chainid.network/
+export const CHAIN_ID = {
+  ETHEREUM: '1',
```

Comment:
Chain ID is always number type, maybe we can change it here too, wdyt?

Reply:
Changed